<launch>
  <node name="feedback_control" pkg="eeg_tests" type="feedback_control.py" output="screen"/>
  <node name="keyboard_input" pkg="eeg_tests" type="keyboard_input.py" output="screen"/>
  <node name="joint_trajectory_action_server" pkg="baxter_interface" type="joint_trajectory_action_server.py" output="screen"/>
</launch>